wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/graph/CallElementToGraphNode.scala

```diff
@@ -139,8 +145,16 @@ object CallElementToGraphNode {
 
         // No input mapping, add an optional input using the default expression
         case withDefault@OverridableInputDefinitionWithDefault(n, womType, expression, _, _) =>
-          val identifier = WomIdentifier(s"${a.workflowName}.$callName.${n.value}")
-          withGraphInputNode(withDefault, OptionalGraphInputNodeWithDefault(identifier, womType, expression, identifier.fullyQualifiedName.value))
+          val identifier = WomIdentifier(
+            localName = s"$callName.${n.value}",
+            fullyQualifiedName = s"${a.workflowName}.$callName.${n.value}"
+          )
```

Comment:
NB: this change to how identifiers are constructed for inputs was the fix of the original bug. Almost everything else was caused by rippling side-effects of this fix